feature/navigation/drawer/dropdown/src/main/kotlin/net/thunderbird/feature/navigation/drawer/siderail/ui/account/SideRailAccountIndicator.kt

```diff
@@ -18,7 +21,7 @@ internal fun AccountIndicator(
             .defaultMinSize(
                 minHeight = MainTheme.spacings.default,
             ),
-        color = calculateAccountColor(accountColor),
+        color = calculateAccountColor(accountColor.toArgb()),
```

Comment:
I would encapsulate this within a remember function. The `calculateAccountColor` function performs color calculations that it would be good to avoid recalculating whenever a recomposition occurs.

```suggestion
        color = remember(accountColor) {
            calculateAccountColor(accountColor.toArgb())
        },
```

Reply:
I removed the `toArgb()` calculation. But remember would require more changes, as it's not a Composable context.

Reply:
I think that works, thanks